Readme.md

```diff
@@ -4,6 +4,62 @@ _This package is currently under development and not fully tested, please don't
 
 Major Rewrite in process.. Stay tuned
 
+### Minimal Setup
+
+With 2.0 there is no default Event Store configured any more.
+Instead the initial Event Store setup is now expected to be defined in the corresponding package or distribution.
```

Comment:
So, what does that mean for people upgrading? Is this "no default config" really necessary/beneficial? I prefer "zero config" setups, where all things basically work out of the box, but if I want, I can totally tune and reconfigure things to my liking.

Reply:
I had the same impression, but @skurfuerst and @kitsunet convinced me that we should strive for a "library" rather than a "framework" for this very package.
Reason: If you have a global Event Store, it's easy to get started but you run into problems as soon as you install another package that makes use of it.